frontend/src/concepts/pipelines/content/createRun/utils.ts

```diff
@@ -44,3 +45,6 @@ export const isFilledRunFormData = (formData: RunFormData): formData is SafeRunF
   !!formData.params &&
   runTypeSafeData(formData.runType) &&
   runTypeSafeDates(formData.runType);
+
+export const getRunTypeCategoryLabel = (runType: PipelineRunType | null): 'schedule' | 'run' =>
```

Comment:
why not just use an enum here?

Reply:
For the return type or the parameter of the function? If for the return type, I didn't see it necessary to create another type just for these possible strings based on how the results of `getRunTypeCategoryLabel` are used.

Reply:
just that the function could be a an enum or some const. A lot of the other spots in the source code use the 

```tsx
const runTypeCategoryLabel = {
[PipelineRunType.Scheduled]: "schedule",
[PipelineRunType.Triggered]: "run",
}
```

Reply:
Sure, I can map it out instead, although I don't think either approach is necessarily better than the other. 1 uses a util, the other a constant. Basically the same amount of code for both too.

Reply:
FWIW - If you type the map, it can also get TS involved to make sure new enum values trigger a need to update it

Reply:
Do you anticipate more than these 3 run types? If not, I don't see that value add there. Also FWIW, the mapped solution requires adding a fallback value in 3 different places and added 4 extra lines of code. The update has been made though.